pkg/shell/base_index.js

```diff
@@ -940,8 +941,23 @@ function CompiledComponents() {
                     label: cockpit.gettext(info.label) || prop,
                     order: info.order === undefined ? 1000 : info.order,
                     icon: info.icon,
-                    wants: info.wants
+                    wants: info.wants,
+                    keywords: info.keywords || { },
+                    keyword: { score: -1 }
                 };
+
+                // Always first keyword should be page name
+                const general = item.keywords.general || { matches: [] };
+                general.matches.unshift(item.label.toLowerCase());
+                item.keywords.general = general;
+
+                // Also we want keywords to be translated and have higher priority
+                // then content of pages
+                Object.values(item.keywords).map(i => {
+                    i.coef = 3;
+                    i.native = true;
```

Comment:
Wont these stomp over any explicit value you set in the manifest?

Reply:
good point, it should read it from manifest and if not set than use default - sorry, forgot to do this